include/anvill/Specification.h

```diff
@@ -64,7 +65,10 @@ struct ControlFlowTargetList final {
   // is, a given indirect jump may target the same address in multiple different
   // ways (e.g. multiple `case` labels in a `switch` statement that share the
   // same body).
-  std::set<std::uint64_t> target_addresses;
+
+  /// The addresses map a target to a given context mapping.
+  std::map<std::uint64_t, std::map<std::string, uint64_t>> target_addresses;
```

Comment:
Is ordering in the second significant, as compared to the prior unordered_map?

Reply:
Updated it can be unordered